src/main/java/uk/gov/hmcts/reform/civil/utils/HearingUtils.java

```diff
@@ -94,20 +97,24 @@ public static String getHearingDuration(CaseData caseData) {
             case DAY_2:
                 return "2 days";
             default:
-                return "not defined";
+                return null;
         }
     }
 
     public static String formatHearingFee(Fee hearingFee) {
         if (nonNull(hearingFee) && hearingFee.getCalculatedAmountInPence().intValue() > 0) {
-            StringBuilder builder = new StringBuilder(hearingFee.getCalculatedAmountInPence().toString());
-            builder.delete(builder.length() - 2, builder.length());
-            if (builder.length() > 3) {
-                builder.insert(1, ',');
-            }
-            builder.insert(0, '£');
-            return builder.toString();
+            DecimalFormat formatter = new DecimalFormat("£#,###");
+            return formatter.format(hearingFee.getCalculatedAmountInPence().intValue() / 100);
+        }
+        return null;
+    }
+
+    public static String getHearingTimeFormatted(String hearingTime) {
+        if (!isEmpty(hearingTime) && hearingTime.length() == 4) {
+            StringBuilder hearingTimeBuilder = new StringBuilder(hearingTime);
+            hearingTimeBuilder.insert(2, ':');
+            return hearingTimeBuilder.toString();
         }
-        return "";
+        return null;
     }
```

Comment:
Consider the following to be more strict with input values, given the "fuzzy" processing we do:

```
    public static String getHearingTimeFormatted(String hearingTime) {
        if (isEmpty(hearingTime) || hearingTime.length() != 4 || !hearingTime.matches("[0-9]+")) {
            return null;
        }

        StringBuilder hearingTimeBuilder = new StringBuilder(hearingTime);
        hearingTimeBuilder.insert(2, ':');
        return hearingTimeBuilder.toString();
    }
```